src/main/scala/gestalt/api.scala

```diff
@@ -122,8 +143,26 @@ object api extends Toolbox { pkg =>
   }
 
   implicit class UntypedTermTreeOps(tree: TermTree) {
-    def select(name: String): TermTree = Select(tree, name)
     def appliedTo(args: TermTree*): TermTree = Apply(tree, args.toList)
+    def selectType(path: String): TypeTree = {
+      val parts = path.split('.')
+
+      val prefix = parts.init.foldLeft[TermTree](tree) { (prefix, name) =>
+        prefix.select(name)
+      }
+
+      TypeSelect(prefix, parts.last)
```

Comment:
In a type select such as `scala.collection.Set`, only the last part is `TypeSelect`, `scala.collection` is still `Select`.